### DISCUSSION TITLE
How to reenable tree previews in 11.x

### DISCUSSION THREAD

**Discussion by HaaseIT** [pimcore/pimcore#16256] [Q&A / Support] [answered]
In 10.x you had a setting in system settings "disable tree previews". If this was unchecked, previews of assets were shown on mouseover.

The 11.x demo does show the tree preview images, so, the feature is still there, does anyone know how to enable this in 11.x?

**Comment by kingjia90** [comment, marked as answer]
It was moved to ⚙️ > `Appearance & Branding`
![image](https://github.com/pimcore/pimcore/assets/6014195/5c2be2af-a6c8-45b8-947a-1c9fd789efdb)
https://github.com/pimcore/pimcore/blob/a44da1e61c0b0cafb93a77631ae1c459881dcc60/doc/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md?plain=1#L289
